go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -183,6 +183,9 @@ func registerTabletEnvFlags(fs *pflag.FlagSet) {
 	fs.Int64Var(&currentConfig.RowStreamer.MaxMySQLReplLagSecs, "vreplication_copy_phase_max_mysql_replication_lag", 43200, "The maximum MySQL replication lag (in seconds) that can exist on a vstreamer (source) before starting another round of copying rows. This helps to limit the impact on the source tablet.")
 
 	fs.BoolVar(&currentConfig.EnableViews, "queryserver-enable-views", false, "Enable views support in vttablet.")
+
+	flagutil.DualFormatBoolVar(fs, &currentConfig.EnablePerWorkloadTableMetrics, "enable-per-workload-table-metrics", defaultConfig.EnablePerWorkloadTableMetrics, "If true, query counts and query error metrics include a label that identifies the workload")
+	flagutil.DualFormatStringVar(fs, &currentConfig.WorkloadLabel, "workload-label", defaultConfig.WorkloadLabel, "The label looked for in query comments to identify the workload. Used in conjunction with enable-per-workload-table-metrics")
```

Comment:
I believe the idea here is that the label itself is customizable. If your query comment has `workload: web-app`, then the `workload-label` flag value would be `workload`. If you want to use `xyzzy` as the label, you can do that too. The other option is to hardcode the label as `workload` or some such pre-defined label. @ejortegau's approach is more flexible at the cost of adding another flag, so that's the trade-off.

Reply:
Correct, I would like to have the label user-customizable as different folks/companies may already be sending some for of this with different labels; or may refer to this in different ways..